lib/l10n/app_es.arb

```
@@ -190,5 +190,14 @@
   "signerErrorRequestInProgress": "Hay otra solicitud en curso. Por favor, espera.",
   "signerErrorNoActivity": "No se pudo abrir el signer. Por favor, inténtalo de nuevo.",
   "signerErrorLaunchError": "Error al abrir la app del signer.",
-  "signerErrorUnknown": "Ocurrió un error con el signer. Por favor, inténtalo de nuevo."
+  "signerErrorUnknown": "Ocurrió un error con el signer. Por favor, inténtalo de nuevo.",
+  "carouselPrivacyTitle": "Privacidad y seguridad",
+  "carouselPrivacyDescription": "Mantén tus conversaciones privadas. Incluso en caso de una brecha, tus mensajes permanecen seguros.",
+  "carouselIdentityTitle": "Elige tu identidad",
+  "carouselIdentityDescription": "Chatea sin revelar tu número de teléfono o correo electrónico. Elige tu identidad: nombre real, seudónimo o anónimo.",
+  "carouselDecentralizedTitle": "Descentralizado y sin permisos",
+  "carouselDecentralizedDescription": "Ninguna autoridad central controla tu comunicación, sin permisos necesarios, sin censura posible."
+,
```

RuboCop (1.84.0)

[fatal] unexpected token tCOMMA
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)